independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -81,405 +123,519 @@
+		  "type": "array",
+		  "items": {
+			"type": "object",
+			"properties": {
+			  "@odata.type": {
+				"type": "string",
+				"description": "Owner object type (eg User or Service Principal)"
+			  },
+			  "id": {
+				"type": "string",
+				"description": "The unique identifier for the owner"
+			  },
+			  "displayName": {
+				"type": "string",
+				"description": "The display name for the service principal"
+			  },
+			  "businessPhones": {
+				"type": "array",
+				"items": {},
+				"description": "The telephone numbers for the user (user type only)"
+			  },
+			  "givenName": {
+				"type": "string",
+				"description": "The given name (first name) of the user (user type only)"
+			  },
+			  "mail": {
+				"type": "string",
+				"description": "The email of the user (user type only)"
+			  },
+			  "mobilePhone": {
+				"type": "string",
+				"description": "The mobile phone number of the user (user type only)"
+			  },
+			  "surname": {
+				"type": "string",
+				"description": "The user's surname (user type only)"
+			  },
+			  "userPrincipalName": {
+				"type": "string",
+				"description": "The user principal name (UPN) of the user  (user type only)"
+			  }
+			}
+		  }
+		}
+	  }
+	},
+	"Application_Definition": {
+		"type": "object",
+		"properties": {
+		  "id": {
+			"type": "string",
+			"description": "The unique identifier for the application"
+		  },
+		  "deletedDateTime": {
+			"type": "string",
+			"description": "The date and time the application was deleted. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time."
+		  },
+		  "appId": {
+			"type": "string",
+			"description": "The unique identifier for the application that is assigned to an application by Azure AD"
+		  },
+		  "applicationTemplateId": {
+			"type": "string",
+			"description": "Unique identifier of the applicationTemplate"
+		  },
+		  "disabledByMicrosoftStatus": {
+			"type": "string",
+			"description": "Specifies whether Microsoft has disabled the registered application"
+		  },
+		  "createdDateTime": {
+			"type": "string",
+			"description": "The date and time the application was registered. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time"
+		  },
+		  "displayName": {
+			"type": "string",
+			"description": "The display name for the application"
+		  },
+		  "description": {
+			"type": "string",
+			"description": "The description for the application"
+		  },
+		  "groupMembershipClaims": {
+			"type": "string",
+			"description": "The groups claim issued in a user or OAuth 2.0 access token that the application expects"
+		  },
+		  "identifierUris": {
+			"type": "array",
+			"items": {
+			  "type": "string"
+			},
+			"description": "The URIs that identify the application within its Azure AD tenant, or within a verified custom domain if the application is multi-tenant"
+		  },
+		  "isDeviceOnlyAuthSupported": {

```

Comment:
@paulculmsee The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please define the "type'. Or please confirm that no users will be affected by this change.

Reply:
I could not pinpoint what you are referring to here sorry. I abstracted the schema into a separate definition for all actions, and there was a couple of erroneous ones that only manifested at scale

Reply:
@Amjed-Ayoub  are you referring to schema changes that paul made to correct the original errors, or what sections specifically? These changes were fixes.

Reply:
Hello @paulculmsee,

this property used to have a type of string. please see below
![image](https://user-images.githubusercontent.com/97551789/218851888-02b76d3a-747a-4716-8b0b-c14ebc067e8a.png)

In this update there is no type defined.